### PR TITLE
made changes to landing pages

### PR DESCRIPTION
**Pull request by Arfah**
I've indented stuff out as it's a temporary change about the QardHasan fellowship so it's easier to make it live when we run the next set of fellowships.  Changed the copy for the fridgeraiders landing page as competition is closed but it's linked from various youtubers videos.

### CODE REVIEW

File: source/fridge-raiders.html.haml [hunk 1]
```diff
@@ -2,44 +2,13 @@
...
-            %p
-              %input{type: "submit", value: "Start Coding Now"}
-
-%hr
-
+      %h2 Learn to code in 12 weeks
+      %h4.subheader Competition is now closed
+      %p  Makers Academy is Europe’s leading Web Developer bootcamp. Mattessons Fridge Raiders sponsored one lucky person to do the course for free! Unfortunately the competition is now closed.
+      %p If you are still interested in becoming a developer, feel free to submit an application and explore the website! Sign up to newsletter to recieve a free Learn Ruby in 15 minutes e-book.
+    = partial :"email-capture", locals: { button_text: "Send my e-book", segments: "website-homepage-lr15" }
```
Comment (nikeshashar) [inline source/fridge-raiders.html.haml]: @Arfah you'd need to change the segment here to be `fridge-raiders` so we can ensure that they get the right emails

File: source/fridge-raiders.html.haml [hunk 2]
```diff
@@ -2,44 +2,13 @@
...
-          .button-row
-            %p
-              %input{type: "submit", value: "Start Coding Now"}
-
-%hr
-
+      %h2 Learn to code in 12 weeks
+      %h4.subheader Competition is now closed
+      %p  Makers Academy is Europe’s leading Web Developer bootcamp. Mattessons Fridge Raiders sponsored one lucky person to do the course for free! Unfortunately the competition is now closed.
+      %p If you are still interested in becoming a developer, feel free to submit an application and explore the website! Sign up to newsletter to recieve a free Learn Ruby in 15 minutes e-book.
```
Comment (nikeshashar) [inline source/fridge-raiders.html.haml]: > recieve

Typo

File: source/payment.html.haml [hunk 3]
```diff
@@ -21,11 +21,11 @@
...
-%section.backgrounded.no-min-height
-  .container
-    %h4
-      For a limited time we're offering a fellowship to a limited number of applicants. Find out more
-      = link_to "here", "/qardhasan"
+-# %section.backgrounded.no-min-height
+-#   .container
+-#     %h4
+-#       For a limited time we're offering a fellowship to a limited number of applicants. Find out more
+-#       = link_to "here", "/qardhasan"
```
Comment (nikeshashar) [inline source/payment.html.haml]: Let's remove the code - it's easy enough to add back later.

File: source/qardhasan.html.haml [hunk 4]
```diff
@@ -5,18 +5,20 @@
   .container
     .centered-row
       %h2 QardHasan Fellowship
-      %h4.subheader Learn to code in 12 weeks from home
-      %p Makers Academy is Europe's leading bootcamp. We've teamed up with Qardhasan to deliver a limited amount of interest free loans for our remote course for anyone from a black, asian, ethnic background.
+      %h4.subheader Currently Closed.
+      %p Makers Academy is Europe's leading bootcamp. We've teamed up with Qardhasan to deliver a limited amount of interest free loans for our remote course for anyone from a black, asian, ethnic background. Sign up to our newsletter to stay informed about future fellowships. 
+    = partial :"email-capture", locals: { button_text: "Sign Up", segments: "website-homepage-lr15" }
```
Comment (nikeshashar) [inline source/qardhasan.html.haml]: change the segment here to be `qardhasan` so we can ensure that they get the right emails

Comment (nikeshashar) [inline source/qardhasan.html.haml]: This would probably need a new campaign / segment too in customer.io

File: source/qardhasan.html.haml [hunk 5]
```diff
@@ -5,18 +5,20 @@
...
-  (function(){var qs,js,q,s,d=document,gi=d.getElementById,ce=d.createElement,gt=d.getElementsByTagName,id='typef_orm',b='https://s3-eu-west-1.amazonaws.com/share.typeform.com/';if(!gi.call(d,id)){js=ce.call(d,'script');js.id=id;js.src=b+'widget.js';q=gt.call(d,'script')[0];q.parentNode.insertBefore(js,q)}})();
+-# = partial :"_qardhasan_info"
+-#
+-# = partial :"vlogs"
+-#
+-# %section.typeform
+-#   .typeform-widget{"data-text" => "Qardhasan Fellowship", "data-url" => "https://makersacademy.typeform.com/to/eMb5ws"}
+-#
+-# :javascript
+-#   (function(){var qs,js,q,s,d=document,gi=d.getElementById,ce=d.createElement,gt=d.getElementsByTagName,id='typef_orm',b='https://s3-eu-west-1.amazonaws.com/share.typeform.com/';if(!gi.call(d,id)){js=ce.call(d,'script');js.id=id;js.src=b+'widget.js';q=gt.call(d,'script')[0];q.parentNode.insertBefore(js,q)}})();
```
Comment (nikeshashar) [inline source/qardhasan.html.haml]: Let's remove this code if it's not being used. We can go back and add what we need at a later date.

File: source/qardhasan.html.haml (hunk 4 above)
Comment (Arfah) [inline source/qardhasan.html.haml]: Nope the idea is they get the normal newsletter and in that we'll eventually say when we do it again. Don't have time to set up a whole new campaign

File: source/fridge-raiders.html.haml (hunk 1 above)
Comment (Arfah) [inline source/fridge-raiders.html.haml]: Nope! There is a reason why  i changed it to the normal one it's so people can be added to the normal mailing list.

**Comment by nikeshashar** [comment]
@rachelcdavies - the segment is still incorrect for the emails. I'm waiting to fix this with @alexandergimson tomorrow - please don't merge

**Comment by nikeshashar** [comment]
Nik to meet with Alex and close on Monday (05 June)

**Comment by nikeshashar** [comment]
I've asked @alexandergimson for stats on each landing page and if we want to keep it or not. If there are not enough hits then it's not worth setting up a new email campaign for each and we should kill it off.

**Comment by nikeshashar** [comment]
Re-done on #924

**Pull request closed without merging**